helpers/seed.go

```diff
@@ -0,0 +1,114 @@
+package helpers
```

Comment:
@Revolyssup why do any of this here? If we already are being explicit that this is a functionality of none provider then add a new method to `none` provider? This approach will result in the following:
1. You don't to keep a global variable around to keep track of the provider
2. You can use the meshkit logger

Reply:
@utkarsh-pro  Can I leave the helper function here? which fetched the files from file system and returns them as a string.
The function which is called can then be a method of local provider.

Reply:
@utkarsh-pro  Then again, how do I keep track of the uuids of seeded components?

Reply:
Actually cant leave helper functions there due to import cycle.

Reply:
Can I add the SeededUUID's field in localProvider struct?

Reply:
You don't have to keep track of the generated UUIDs in the provider. Here are rough function signatures:
```go
// cmd/main.go
// ...
seededUUIDs, err := provider.SeedContent()
// ...
<- c 
// Clean up
provider.CleanupSeeded(seededUUIDs)
```